install.sh

```diff
@@ -7,10 +7,13 @@
 
 set -euo pipefail
 
+NO_HOOK_NAME="no-hook"
+
 DEBUG=${DEBUG:-''}
 HOOK_NAME="${1:-pre-push}"
 case "$HOOK_NAME" in
 pre-commit | pre-push) REPO_HOOK_TARGET=".git/hooks/${HOOK_NAME}" ;;
+"$NO_HOOK_NAME") REPO_HOOK_TARGET="";;
 *)
```

Comment:
Could you explain what this line is for? "$NO_HOOK_NAME") REPO_HOOK_TARGET="";;

Reply:
Hi there - yes, the idea is to skip the `*) exit 1 ` that we see starting on line 17, and also to explicitly set the REPO_HOOK_TARGET as empty (so as to ensure that no value has been otherwise set)

Arguably that's me being overly cautious, but also me trying to make the intent of the option clear for what follows in the code

Reply:
(possibly I also intended at one point to check for an empty string on line 248 instead of testing for `"$HOOK_NAME" == "$NO_HOOK_NAME"` and changed direction later)

Reply:
Gotcha. Thanks for clarifying!